src/test/db.rs

```diff
@@ -41,7 +40,7 @@ async fn get_coll_info(db: &Database, filter: Option<Document>) -> Vec<Collectio
 #[tokio::test]
 #[function_name::named]
 async fn list_collections() {
-    let client = TestClient::new().await;
+    let client = Client::test_builder().build().await;
```

Comment:
WDYT about making `build` an `IntoFuture` impl instead? It'd be nice to make the no-configuration case a little less verbose, although I'm not sure what the right name for `test_builder` would be if there's no corresponding `build`.

Reply:
Ah, that cleans things up nicely :)  Renamed the method to `Client::for_test` which (IMO) reads well both with and without chained methods.